packages/ui/src/Wrapper.tsx

```diff
@@ -18,7 +20,7 @@ export const Wrapper = memo(function Wrapper({
       alignItems='center'
     >
       <Box
-        maxW='960px'
+        maxW={size === 'lg' ? '1200px' : '960px'}
```

Comment:
So `md` is just 960px?

Reply:
Yep, or more literally, anything not `lg` is `960px`. But the prop is typed as `'lg' | 'md'` so in practice that means `md` is `960px`